node/node.go

```diff
@@ -1432,6 +1436,10 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		service := grpcserver.NewPostService(app.addLogger(PostServiceLogger, lg).Zap())
 		app.grpcServices[svc] = service
 		return service, nil
+	case grpcserver.PostInfo:
+		service := grpcserver.NewPostInfoService(app.addLogger(PostInfoServiceLogger, lg).Zap(), app.atxBuilder)
```

Comment:
Would it make sense to pass `postStates` here instead of `app.atxBuilder`?

Reply:
Yes, but it would require making more code use the `types.IdentityDescriptor` because the `PostInfoService` needs to map `types.IdentityDescriptor` -> `types.PostState` (not `types.NodeID`).  I think about changing `Nipostbuilder::Proof` to use it (instead of NodeID) so that it can log the `name` too in ` post service not connected - waiting for reconnection   {"service id": "5ea2c9975a185eeaa40bc9351297aa98c85952553f0fde4707ad783943c65e2e", "error": "post service not registered"}` in a follow up.